recipes/m2w64-muparser/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "2.2.5" %}
+
+package:
+  name: m2w64-muparser
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/beltoforion/muparser/archive/v{{ version }}.tar.gz 
+  md5: 02dae671aa5ad955fdcbcd3fee313fb7
+  patches:
+    - no-console.patch
+
+build:
+  number: 0
+  skip: true  # [not win]
+
+requirements:
+  build:
+    - m2w64-toolchain
+    - cmake
+
+test:
+  requires:
+    - python {{ os.environ.get('PY_VER', '') + '*' }}
```

Comment:
This 'use ctypes to check the library loads' thing came up before and I'm very against it, to me it's a misuse of Python because it's easy to do.

It should be a goal that conda-forge is bootstrappable without dependency cycles and that precludes using python as a general 'can this shared library be loaded' checker.

Instead we should write simple C programs that are distributed as a part of conda-build or as a separate shared-library-load checking program. conda-build has pyldd now and these checking efforts would be better spent improving that so that it can handle Windows.